MarsRover/InputCommands.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace MarsRover
+{
+    public class InputCommands
+    {
+        private string[] _commands = { "L", "R", "F", "B" };
+        private string _input;
+
+        public InputCommands(string input) => _input = input.ToUpper();
+
+        public string ValidateAndReturnCommand()
+        {
+            if (ValidCommandLetter(_input))
+            {
+                switch (_input)
+                {
+                    case ("L"):
+                        return "MarsRover.TurnLeftCommand";
+                    case ("R"):
+                        return "MarsRover.TurnRightCommand";
+                    case ("F"):
+                        return "MarsRover.MoveForwardCommand";
+                    case ("B"):
+                        return "MarsRover.MoveBackwardCommand";
+                }
+            }
+
+            return "MarsRover.InvalidCommand";
+        }
+
+        private bool ValidCommandLetter(string input) => Array.Exists(_commands, commandLetter => commandLetter == input);
```

Comment:
By convention, it's nice to prepend the name functions like this with "Is."  `IsValidCommandLetter`

By doing it this way, when using this function it reads more fluently.